src/screens/Asset/Asset.js

```diff
@@ -155,207 +166,204 @@ const SyntheticAssetIcon = styled(Image)`
+            onPressExchange={goToExchangeFlowIfAvailable}
+            noBalance={isWalletEmpty}
+            isSendDisabled={!isSendActive}
+            isReceiveDisabled={!isReceiveActive}
+            showButtons={isSynthetic ? ['receive'] : undefined} // eslint-disable-line i18next/no-literal-string
           />
-          <DataWrapper>
-            <ValueWrapper>
-              {!!isSynthetic &&
-                <SyntheticAssetIcon source={lightningIcon} />
-              }
-              <TokenValue isSynthetic={isSynthetic}>
-                {t('tokenValue', { value: displayAmount, token })}
-              </TokenValue>
-            </ValueWrapper>
-            {!!isListed &&
-              <ValuesWrapper>
-                <ValueInFiat>
-                  {fiatAmount}
-                </ValueInFiat>
-              </ValuesWrapper>
-            }
-            {!isListed &&
-            <Disclaimer>
-              {disclaimer}
-            </Disclaimer>
-            }
-          </DataWrapper>
-          <AssetCardWrapper>
-            <AssetButtons
-              onPressReceive={this.openAddFundsModal}
-              onPressSend={this.goToSendTokenFlow}
-              onPressExchange={isSupportedByExchange ? () => this.goToExchangeFlow(token) : null}
-              noBalance={isWalletEmpty}
-              isSendDisabled={!isSendActive}
-              isReceiveDisabled={!isReceiveActive}
-              showButtons={isSynthetic ? ['receive'] : undefined} // eslint-disable-line i18next/no-literal-string
-            />
-            {!isSendActive && <SWActivationCard />}
-          </AssetCardWrapper>
-
-          {!!relatedTransactions.length && (
-            <ActivityFeed
-              feedTitle={t('title.transactions')}
-              navigation={navigation}
-              feedData={relatedTransactions}
-              isAssetView
-            />
-          )}
-
-          {!!historyItems.length && (
-            <HistoryList items={historyItems} />
-          )}
-        </ScrollWrapper>
-        <RetryGraphQueryBox
-          message={t('error.theGraphQueryFailed.isTokenSupportedByUniswap')}
-          hasFailed={!isSupportedByExchange && uniswapTokensGraphQueryFailed}
-          isFetching={isFetchingUniswapTokens}
-          onRetry={getExchangeSupportedAssets}
-        />
-      </ContainerWithHeader>
-    );
-  }
-}
+          {!isSendActive && <SWActivationCard />}
+        </AssetCardWrapper>
+        {!!transactions.length && (
+          <>
+            {isArchanovaAccount(activeAccount) && (
+              <ActivityFeed
+                feedTitle={t('title.transactions')}
+                navigation={navigation}
+                feedData={transactions}
+                isAssetView
+              />
+            )}
+            {/* $FlowFixMe: should be fine after Archanova history mappings are discarded */}
+            {isEtherspotAccount(activeAccount) && <HistoryList items={transactions} />}
```

Comment:
👍🏻 
There is a similar history section in collectible details, could you add etherspot history list data there as well?

Reply:
Will add include into multi chain history PR.